README.md

```diff
@@ -1,2 +1,189 @@
-# Hellow-World
-My fist module one task
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="assets\images\cvimage2.JPG" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello World Project</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+>
+
+**[Hello World]** is a project to understand the basics of Github flow and lint rules
+
+## 🛠 Built With <a name="HTML/CSS"></a>
+
+### Tech Stack <a name="HTML/CSS"></a>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[index.html file]**
+- **[style.css file]**
+- **[.gitignore file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+- [Live Demo Link](https://google.com)
+
```

Comment:
- [ ] Kindly make sure to remove the `Live Demo` link you added because it is not the deployed version of this project so it is better to remove the section completely or you add the message status like `Live Demo Coming Soon` instead of adding the wrong link while visitor expects to the deployed version there. Kindly fix that.

Reply:
> * [ ]  Kindly make sure to remove the `Live Demo` link you added because it is not the deployed version of this project so it is better to remove the section completely or you add the message status like `Live Demo Coming Soon` instead of adding the wrong link while visitor expects to the deployed version there. Kindly fix that.

- [ ] As said by the previous reviewer, you need to remove the live demo because it is not the live version and you can just add coming soon instead of adding a link that does not fit in.